airflow/jobs/scheduler_job.py

```diff
@@ -1454,39 +1453,11 @@ def _do_scheduling(self, session) -> int:
             if settings.USE_JOB_SCHEDULE:
                 self._create_dagruns_for_dags(guard, session)
 
-            dag_runs = self._get_next_dagruns_to_examine(session)
+            self._start_dagrun(guard, session)
```

Comment:
```suggestion
            self._start_queued_dagrun(session)
            guard.commit()
```

(as per my other comments)